frontend/components/modals/ExecuteWithOtherWalletModal.vue

```diff
@@ -37,9 +33,7 @@
                 <div
                     v-if="!isOutcomeWalletInputValueValid && outcomeWalletInputValue"
                     class="flex flex-col space-y-3 my-3 md:mb-0"
-                >
-                    <Alert type="error" :message="`Address ${outcomeWalletInputValue} seems to be incorrect`" />
-                </div>
+                ></div>
```

Comment:
The wrapping div is the leftover, in case of the error, it will add margin of 6 under the input